File: app/src/main/java/club/hutcwp/coolweather/util/HttpUtil.java
/*
 * Copyright (c) 2017. The Android Project Create By Hutcwp.
 */

package club.hutcwp.coolweather.util;

import okhttp3.OkHttpClient;
import okhttp3.Request;


/**
 * Created by hutcwp on 2017/4/9.
 * Mail : hutcwp@foxmail.com
 * Blog : hutcwp.club
 * GitHub : github.com/hutcwp
 */

public class HttpUtil {

    public static void sendOkHttpRequest(String address,okhttp3.Callback callback){

        OkHttpClient client = new OkHttpClient();
        Request request = new Request.Builder().url(address).build();
        client.newCall(request).enqueue(callback);


    }

}

File: app/src/main/java/club/hutcwp/coolweather/util/HttpUtil.kt
/*
 * Copyright (c) 2017. The Android Project Create By Hutcwp.
 */

package club.hutcwp.coolweather.util

import okhttp3.OkHttpClient
import okhttp3.Request


/**
 * Created by hutcwp on 2017/4/9.
 * Mail : hutcwp@foxmail.com
 * Blog : hutcwp.club
 * GitHub : github.com/hutcwp
 */

object HttpUtil {

    fun sendOkHttpRequest(address: String, callback: okhttp3.Callback) {
        val client = OkHttpClient()
        val request = Request.Builder().url(address).build()
        client.newCall(request).enqueue(callback)
    }
}
